docs/app/services/codepen/codepen.service.ts

```diff
@@ -161,6 +161,7 @@ export class CodePenService {
 
     // Script files for CodePen to reference
     private CODEPEN_SCRIPTS = [
+        'https://unpkg.com/core-js@2.5.1/client/core.js',
```

Comment:
This will fix the codepen, however teams consuming only the ux-aspects-ng1.js script file will still get this error in IE if they try to use this feature I think?

Reply:
Okay, I thought corejs was one of our dependencies

Reply:
It is, but its only included in our Angular 4 stuff, so it may not be included if they are only using our ng1 script